.github/workflows/build-and-deploy.yml

```diff
@@ -98,7 +98,7 @@ jobs:
           echo "MINOR=$MINOR"
           echo "PATCH=$PATCH"
 
-          docker build . --file Dockerfile \
+          DOCKER_BUILDKIT=1 docker build . --file Dockerfile \
```

Comment:
Might be cleaner if you just do

```
export DOCKER_BUILDKIT=1
docker build ...

```

I think that's easier to read, and means if we do other docker builds later.

The way we do this in other projects is have a `just docker-build` command that takes care of any environment variables, and use that in the gh workflow.

Reply:
Thanks @bloodearnest. I don't want to extend this to editing the workflow too much, but have made the change you suggested in 7fe47369.